stripe/src/test/java/com/stripe/android/model/PaymentIntentFixtures.kt

```diff
@@ -680,4 +680,70 @@ internal object PaymentIntentFixtures {
         }
         """.trimIndent()
     )
+
+    val ALIPAY_REQUIRES_ACTION = JSONObject(
```

Comment:
add `_JSON` suffix to this name to clarify that it's a JSON object

Reply:
done